src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -396,7 +409,14 @@ private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         volumes.add(new VolumeBuilder().withName(WORKSPACE_VOLUME_NAME).withNewEmptyDir("").build());
         Map<String, Container> containers = new HashMap<>();
 
+        if (template.isCustomJnlpContainerEnabled() && template.getJnlpContainer() != null) {
```

Comment:
No, the pull request does not provide a way to do so. It provides a more clear way to define the which container will act as a `jnlp` container and also adds the inheritance thing, but no way to explicitly request `no jnlp container` at all.

Even, if we add an option for that. I think that it wouldn't work, because you can't have a slave pod without a jnlp container. 

If you are using a hybrid container (e.g. build tools + jnlp), it is still a jnlp container, so it could be defined in the jnlp container field. 

Having no jnlp container at all could possibly make sense only in the case where you already had a slave pod running and for some reason you needed an additional container to execute some steps in the pipeline. Of course, this is still not supported (yet) by the plugin (because we would need to have a way to share the workspace, #114 might help in that regard).

Now, if you still feel there is a demand for such option, I don't mind adding it as part of this pull request.